content/appendices/symbol-lookup-cheatsheet.md

```diff
@@ -40,7 +39,7 @@ Here is a more elaborate explanation of Pony's use of special characters: (a lin
 |    | (2) middle of line: method call |
 | `()` | (1) parentheses, for function or behavior parameters |
 |    | (2) making a tuple (values separated by `,`) |
-|    | (3) making an enumeration (values separated by <code>&#124;</code>) |
+|    | (3) making an enumeration (values separated by `&#124;` |
```

Comment:
```suggestion
|    | (3) making an enumeration (values separated by `|`) |
```

Reply:
I assume that it isn't a `|` because that might break formatting. I'd rather not change that as part of this PR.

Reply:
I see it does. I'll add your change and we can see what it looks like.

Reply:
As it is, it already breaks formatting (and also is missing a parentheses). See [Netlify's preview](https://deploy-preview-455--pedantic-mahavira-2a84e1.netlify.app/appendices/symbol-lookup-cheatsheet.html):

![image](https://user-images.githubusercontent.com/3129194/103295628-20136080-49d3-11eb-99a4-c688e2929609.png)

![image](https://user-images.githubusercontent.com/3129194/103295579-0a05a000-49d3-11eb-8bed-4cb18932d910.png)

I imagine that using the backticks should be enough for Markdown to not consider it as a formatting character.